src/blueapi/cli/cli.py

```diff
@@ -258,6 +259,51 @@ def stop(obj: dict) -> None:
     pprint(client.cancel_current_task(state=WorkerState.STOPPING))
 
 
+@controller.command(name="env")
+@check_connection
+@click.option(
+    "-r",
+    "--reload",
+    is_flag=True,
+    type=bool,
+    help="Reload the current environment",
+    default=False,
+)
+@click.pass_obj
+def env(obj: dict, reload: Optional[bool]) -> None:
+    """
+    Inspect or restart the environment
+    """
+
+    client: BlueapiRestClient = obj["rest_client"]
+    if not reload:
+        pprint(client.get_environment())
+        return
+
+    # Reload the environment if needed
+    print("Reloading the environment...")
+    pprint(client.reload_environment())
+
+    # Initialize a variable to keep track of the environment status
+    environment_initialized = False
+
+    # Use a while loop to keep checking until the environment is initialized
+    while not environment_initialized:
+        # Fetch the current environment status
+        environment_status = client.get_environment()
+
+        # Check if the environment is initialized
+        if environment_status.initialized:
+            print("Environment is initialized.")
+            environment_initialized = True
+        else:
+            print("Waiting for environment to initialize...")
+            sleep(5)  # Wait for 5 seconds before checking again
```

Comment:
I think 5 seconds is a long time, would rather check every 1 second 10 times, say, and then raise a `TimeoutError`

Reply:
ok